packages/wrangler/src/user.tsx

```diff
@@ -661,8 +661,27 @@ async function exchangeRefreshTokenForAccessToken(): Promise<AccessContext> {
       "Content-Type": "application/x-www-form-urlencoded",
     },
   });
+
   if (response.status >= 400) {
-    throw await response.json();
+    let tokenExchangeResErr = undefined;
+
+    tokenExchangeResErr = await response.text();
```

Comment:
```suggestion
    let tokenExchangeResErr = await response.text();
```

Reply:
Or alternatively move the `await response.text()` into the try-catch block?

Reply:
Test still passes so done lol